backendservice/src/main/java/com/google/gpay/virtualqueue/backendservice/entities/Token.java

```diff
@@ -0,0 +1,33 @@
+package com.google.gpay.virtualqueue.backendservice.entities;
+
+public class Token {
+    private Integer shopId;
+    private Integer tokenNo;
+    private Boolean status;
+
+    public Token(){
+    }
+    public Token(Integer shopId, Integer tokenNo, Boolean status){
```

Comment:
Can we explore lombok @Data annotation for this as well?

Reply:
okay I will look into that.

Reply:
done!